src/arrayMethodSort.js

```diff
@@ -1,11 +1,39 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+function toStringSort(firstElment, seondElement) {
+  const firstStrElement = String(firstElment);
+  const secondStrElement = String(seondElement);
+
+  if (firstStrElement > secondStrElement) {
+    return 1;
+  } else if (firstStrElement < secondStrElement) {
+    return -1;
+  } else {
+    return 0;
+  }
```

Comment:
taking into account that you are doing `return` in each condition better rewrite it without `else if` and `else`
the result will be more readable 
generally: try to avoid `else if` and `else` if it's possible

Reply:
Ok. Thanks